README.rst

```diff
@@ -174,6 +174,12 @@ append-only tables, using the processing power of Google's infrastructure.
 
 This package is still being implemented, but it is almost complete!
 
+.. code:: python
+    from gcloud import bigquery
+    client = bigquery.Client()
+    dataset = client.dataset('dataset_name')
+    dataset.create()  # API request
```

Comment:
I would add a table to the dataset, populate it from a CSV file, and then query it via a synchronous query.  Have a look at #1833 / #1836 for snippets.